pkg/hub/submarineragent/controller.go

```diff
@@ -574,7 +574,8 @@ func newSubmarinerManifestWork(managedCluster *clusterv1.ManagedCluster, config
 	return newManifestWork(SubmarinerCRManifestWorkName, managedCluster.Name, config, submarinerCRFile)
 }
 
-func newOperatorManifestWork(managedCluster *clusterv1.ManagedCluster, config interface{}) (*workv1.ManifestWork, error) {
+func newOperatorManifestWork(managedCluster *clusterv1.ManagedCluster, config *brokerinfo.SubmarinerBrokerInfo,
```

Comment:
Why was the `config` param changed from **interface{}** to ***brokerinfo.SubmarinerBrokerInfo**?

Reply:
To increase type safety, but I shouldn’t have documented it and it’s a separate change, so I’ll roll it back.